content/en/agent/basic_agent_usage/aix.md

```diff
@@ -14,114 +14,86 @@ algolia:
-* **INSECURE**: defaults to `false`. Allows skipping TLS validation.
+### Install Agent
 
-Alternatively, download links for the latest releases can be found on [this page][2].
+To install the Agent on AIX, follow the [in-app instructions in Fleet Automation][6], and run the generated script on your hosts.
 
-The installer may be executed as follows (as root):
+{{< img src="/agent/basic_agent_usage/aix_img_july_25.png" alt="The Datadog Agent installation step for hosts AIX." style="width:90%;">}}
 
-{{< code-block lang="shell" wrap="true" >}}
-installp -aXYgd ./datadog-unix-agent-<VERSION>.bff -e dd-aix-install.log datadog-unix-agent
-{{< /code-block >}}
 
-This installs the Agent in `/opt/datadog-agent`.
 
 ### Installation log files
 
 You can find the Agent installation log in the `dd-aix-install.log` file. To disable this logging, remove the `-e dd-aix-install.log` parameter in the installation command.
 
 ## Commands
 
-| Description                     | Command (as root)           |
-|---------------------------------|-----------------------------|
-| Start Agent as a service        | `startsrc -s datadog-agent` |
-| Stop Agent running as a service | `stopsrc -s datadog-agent`  |
-| Status of Agent service         | `lssrc -s datadog-agent`    |
-| Status page of running Agent    | `datadog-agent status`      |
-| Send flare                      | `datadog-agent flare`       |
-| Display command usage           | `datadog-agent --help`      |
-
-## Configuration
-
-The configuration files and folders for the Agent are located in `/etc/datadog-agent/datadog.yaml`
-
-A sample configuration file can be found in `/etc/datadog-agent/datadog.yaml.example`.
+| Description   | Command (as root)           |
+|---------------|-----------------------------|
+| Start Agent   | `startsrc -s datadog-agent` |
+| Stop Agent    | `stopsrc -s datadog-agent`  |
+| Status of Agent service | `lssrc -s datadog-agent`|
+| Status Agent service | `datadog-agent status`      |
+| Send flare | `datadog-agent flare` |
+| Show all commands | `datadog-agent --help`  |
 
-A basic configuration typically requires your Datadog API key. To submit your metrics to a different site (for example, the EU instance), the `site` configuration option is available.
 
-Occasionally a proxy configuration must be specified depending on your network setup.
+## Configure the Agent
 
-**Configuration files for Integrations:**
-`/etc/datadog-agent/conf.d/`
+The [Datadog Agent configuration file][7] is located in `/etc/datadog-agent/datadog.yaml`. This YAML file holds the host-wide connection details used to send data to Datadog including:
+ * `api_key`: your organization's [Datadog API key][8]  
+ - `site`: target Datadog region (`datadoghq.com`, - `datadoghq.eu`, `us3.datadoghq.com`, `us5.datadoghq.com`, `ddog-gov.com`)  
+* `proxy`: HTTP/HTTPS proxy endpoints for outbound traffic (see [Datadog Agent Proxy Configuration][9])  
+ Default tags, log level, and Datadog configurations.
```

Comment:
It's not clear to me how this line is connected with the proxy configuration. Would it be possible to clarify that here?